doc/poor_mans_mmp_todo.txt

```diff
@@ -0,0 +1,33 @@
+To be added to an Issue and removed from the tree:
+
+Before performing an MMP write, check the spa to see if a sync wrote an
+uberblock to disk within the last zfs_mmp_interval.  If so, skip the MMP write
+as the synced uberblock provides the "heartbeat" we need.  Idea by Andreas
+Dilger.
+
+Reintroduce vdev_disk_physio() for MMP writes, if we find issues with using
+zio pipeline.  It exists in Illumos but was dropped in the port because there
+was no consumer.  It would get to bio_submit() and friends much more directly.
+
+Alter VDEV_UBERBLOCK_SLOT, currently (txg % count), to reduce its cost.  The
+concern is that it is too expensive on 32-bit systems since txg is uint64_t.
```

Comment:
I think this concern is outdated, we're already doing far more expensive things on 32-bit systems for this to matter.